src/constants/base.ts

```diff
@@ -40,6 +40,7 @@ export enum EventTags {
 }
 
 export enum PaymentsProcessors {
+  LNURL = 'lnurl',
```

Comment:
this has been added to settings, but not to `resources/default-settings.yaml`
please add default settings for this payment processor

Reply:
Done!